lbry/schema/mime_types.py

```diff
@@ -166,10 +168,41 @@
     '.wmv': ('video/x-ms-wmv', 'video')
 }
 
+# maps detected extensions to the possible analogs
+# i.e. .cbz file is actually a .zip
+synonyms_map = {
+    '.zip': ['.cbz'],
+    '.rar': ['.cbr'],
+    '.ar': ['.a']
+}
+
+log = logging.getLogger(__name__)
+
 
 def guess_media_type(path):
     _, ext = os.path.splitext(path)
     extension = ext.strip().lower()
+
+    # try detecting real file format if path points to a readable file
+    try:
+        kind = filetype.guess(path)
+        if kind:
+            realext = f".{kind.extension}"
+
+            # override extension parsed from file...
+            if extension != realext:
+                if extension:
+                    log.warning(f"file extension does not match it's contents {path}, identified as {realext}")
+                else:
+                    log.debug(f"file {path} does not have extension, identified by contents as {realext}")
+
+                # don't do anything if extension is in synonyms
+                if not extension in synonyms_map.get(realext, []):
```

Comment:
I think this would be clearer to move the `not` to be before the `in`. Like this:
```python
if extension not in synonyms_map.get(realext, []):
```

Reply:
Adjuster per your feedback.